internal/distro/rhel90/package_sets.go

```diff
@@ -705,6 +705,88 @@ func rhelEc2SapPackageSet(t *imageType) rpmmd.PackageSet {
 	return ec2SapPackageSet
 }
 
+// common GCE image
+func gceCommonPackageSet(t *imageType) rpmmd.PackageSet {
+	ps := rpmmd.PackageSet{
+		Include: []string{
+			"langpacks-en", // not in Google's KS
+			"acpid",
+			"dhcp-client",
+			"dnf-automatic",
+			"net-tools",
+			//"openssh-server", included in core
+			"python3",
+			"rng-tools",
+			"tar",
+			"vim",
+
+			// GCE guest tools
+			"google-compute-engine",
+			"google-osconfig-agent",
+			"gce-disk-expand",
+			// GCP SDK
+			"google-cloud-sdk",
+
+			// Not explicitly included in GCP kickstart, but present on the image
+			// for time synchronization
+			"chrony",
+			"timedatex",
+			// Detected Platform requirements by Anaconda
+			"qemu-guest-agent",
+			// EFI
+			"grub2-tools-efi",
+			"firewalld", // not pulled in any more as on RHEL-8
+		},
+		Exclude: []string{
+			"alsa-utils",
+			"b43-fwcutter",
+			"dmraid",
+			"eject",
+			"gpm",
+			"irqbalance",
+			"microcode_ctl",
+			"smartmontools",
+			"aic94xx-firmware",
+			"atmel-firmware",
+			"b43-openfwwf",
+			"bfa-firmware",
+			"ipw2100-firmware",
+			"ipw2200-firmware",
+			"ivtv-firmware",
+			"iwl100-firmware",
+			"iwl1000-firmware",
+			"iwl3945-firmware",
+			"iwl4965-firmware",
+			"iwl5000-firmware",
+			"iwl5150-firmware",
+			"iwl6000-firmware",
+			"iwl6000g2a-firmware",
+			"iwl6050-firmware",
+			"kernel-firmware",
+			"libertas-usb8388-firmware",
+			"ql2100-firmware",
+			"ql2200-firmware",
+			"ql23xx-firmware",
+			"ql2400-firmware",
+			"ql2500-firmware",
+			"rt61pci-firmware",
+			"rt73usb-firmware",
+			"xorg-x11-drv-ati-firmware",
+			"zd1211-firmware",
+		},
+	}.Append(bootPackageSet(t)).Append(coreOsCommonPackageSet(t)).Append(distroSpecificPackageSet(t))
+
+	// Some excluded packages are part of the @core group package set returned
+	// by coreOsCommonPackageSet(). Ensure that the conflicting packages are
+	// returned from the list of `Include` packages.
+	return ps.ResolveConflictsExclude()
+}
+
+// GCE BYOS image
+func gcePackageSet(t *imageType) rpmmd.PackageSet {
+	return gceCommonPackageSet(t)
+}
```

Comment:
Will the `gcePackageSet` change at some point?  Do we need this?

Reply:
No, but there will be a RHUI variant of the package set which will inherit from `gceCommonPackageSet`, similar as there is in e.g. 8.6:
```go
// GCE BYOS image
func gcePackageSet(t *imageType) rpmmd.PackageSet {
	return gceCommonPackageSet(t)
}

// GCE RHUI image
func gceRhuiPackageSet(t *imageType) rpmmd.PackageSet {
	return rpmmd.PackageSet{
		Include: []string{
			"google-rhui-client-rhel8",
		},
	}.Append(gceCommonPackageSet(t))
}
```

The reason why it does not exist yet is that Google still didn't built a RHUI client packages for RHEL-9.

My reasoning for this piece of code was to:

1. Make a logical split between the BYOS and RHUI images package set.
2. Make the GCE code added by this PR ideally identical in all distro definitions.

I could just use the `gceCommonPackageSet` directly for the BYOS image or rename it to `gcePackageSet`. In such case, the RHUI image would be inheriting the BYOS package set and adding some packages on top of it.

I'm happy to rework this if you find it confusing and / or useless...

Reply:
> My reasoning for this piece of code was to:
> 
> 1. Make a logical split between the BYOS and RHUI images package set.
> 2. Make the GCE code added by this PR ideally identical in all distro definitions.
> 
> ...
> 
> I'm happy to rework this if you find it confusing and / or useless...

Sounds good.  Makes sense.  Let's keep it.